src/server/repositories/user.repository.js

```diff
@@ -1,49 +1,95 @@
-const users= [
-    {
-    id:1,
-    name:"John",
-    age:23
-},
-    {
-    id:2,
-    name:"Peter",
-    age:34
-},
-    {
-    id:3,
-    name:"Anna",
-    age:45
-},
-]
-export const getOneById=(id)=>{
-    return users.find((item)=>item.id==id)
+import sequelize from '../database/connection'
+import User from '../models/User.models'
+
+// const users = [
+//     {
+//         id: 1,
+//         name: "John",
+//         age: 23
+//     },
+//     {
+//         id: 2,
+//         name: "Peter",
+//         age: 34
+//     },
+//     {
+//         id: 3,
+//         name: "Anna",
+//         age: 45
+//     }
+// ]
+export const getOneById =async (id) => {
+    return users.find((item) => item.id == id)
 }
-export const getAll=()=>{
-    return users
+export const getAll = async () => {
+    // User.init({
+    //     // Model attributes are defined here
+    //     id: {
+    //       type: DataTypes.INTEGER,
+    //       autoIncrement: true,
+    //       primaryKey: true
+    //     },
+    //     name: {
+    //       type: DataTypes.STRING,
+    //       allowNull: false
+    //     },
+    //     age: {
+    //       type: DataTypes.INTEGER
+    //       // allowNull defaults to true
+    //     }
+    //   }, {
+    //     // Other model options go here
+    //     sequelize, // We need to pass the connection instance
+    //     modelName: 'User' // We need to choose the model name
+    //   });
+    // const jane = await User.create({ name: "Jane", age: 12 });
+     const users2 = await sequelize.query('SELECT * FROM users')
```

Comment:
`users2` is not a really good name for a variable.
Use `User` model instead of `sequelize.query`